Makefile

```diff
@@ -243,6 +240,7 @@ build-docs-preview:
 generate-schemas:
 	go run hack/schemas/main.go
 
-.PHONY: generate-statik
-generate-statik:
+# static files
+
+$(STATIK_FILES): go.mod docs/content/en/schemas/*
```

Comment:
Why do we need add the dependency to go.mod and docs/ here? These files are checked in the repository and would always be present

Reply:
The dependency tells Make which inputs it should look at to decide if the output needs to be rebuilt.

Reply:
Make sense. So statik would only run if any of those inputs to `hack/generate_static.go` change